themes/jb/layouts/calendar/single.html

```diff
@@ -320,24 +326,30 @@ <h1>{{.Title}}</h1>
           <option value="Pacific/Tongatapu">(GMT+13:00) Tongatapu</option>
           <option value="Pacific/Kiritimati">(GMT+14:00) Kiritimati</option>
         </select>
-    </div>
-  </div>
 
-  <script>
-    var calendarBase = document.querySelector('iframe').dataset.src;
+        <br>
+        <br>
 
-    // Detect timezone automatically
-    var tz = Intl.DateTimeFormat().resolvedOptions().timeZone;
-    var tzname = encodeURIComponent(tz);
-    document.querySelector('iframe').src = calendarBase + "&ctz=" + tzname;
+        <div class="container">
+          <div class="columns is-multiline">
+             {{ range where .Site.Pages  "Section" "show" }}
+              {{ if (isset .Params "live") }}
+                <div class="column is-6 is-3-fullhd is-3-desktop is-12-mobile"  style="display: flex;">
+                  {{ partial "episode/schedule.html" . }}
+                </div>
+              {{ end }}
+            {{ end }}
+          </div>
+        </div>
 
-    // Change on timezone after dropdown
-    document.querySelector('#ctz').onchange = function () {
-      var tzname = document.querySelector('#ctz').value;
-      
-      if(tzname && tzname != "none") {
-        document.querySelector('iframe').src = calendarBase + "&ctz=" + tzname;
-      }
-    };
-  </script>
+        <br>
+        <br>
+
+        Subscribe To The Calendar: <a href="{{.Site.Params.calendar.rss}}">iCalendar</a>
+
+        <iframe frameborder="0" scrolling="no" data-src="//www.google.com/calendar/embed?showTitle=0&amp;showNav=0&amp;showPrint=0&amp;showTabs=0&amp;showCalendars=0&amp;mode=AGENDA&amp;wkst=1&amp;bgcolor=%23FFFFFF&amp;src={{.Site.Params.calendar.embedd}}&amp;color=%235229A3" width="100%" height="400px"></iframe>
```

Comment:
So personally I try to avoid `<br/>` for spacing content and untagged text in my html. I would use the classes provided by [Bulma](https://bulma.io/documentation/helpers/spacing-helpers/) to handle that with CSS. 

I am thinking this is fine for this PR. But a follow up (of either a calendar rework using or of a styling change) could be where this will happen.

Reply:
I added a div and some Bulma classes and removed the `<br>`s.